perllib/FixMyStreet/App/Form/Waste/Garden.pm

```diff
@@ -122,13 +133,16 @@ has_field current_bins => (
     range_end => 3,
 );
 
-has_field new_bins => (
+has_field bins_wanted => (
     type => 'Integer',
-    label => 'Additional bins required (0-3)',
+    label => 'Total number of bins required (0-3)',
     tags => { number => 1 },
     required => 1,
     range_start => 0,
     range_end => 3,
+    tags => {
```

Comment:
This is overriding the previous tags => number => 1 (whatever that is doing :) )